pay-api/src/pay_api/services/fas/routing_slip_status_transition_service.py

```diff
@@ -43,8 +43,7 @@ class RoutingSlipStatusTransitionService:  # pylint: disable=too-many-instance-a
         RoutingSlipStatus.HOLD.value: [
             RoutingSlipStatus.ACTIVE.value,
             RoutingSlipStatus.NSF.value,
-            RoutingSlipStatus.VOID.value,
-            RoutingSlipStatus.CORRECTION.value
+            RoutingSlipStatus.VOID.value
```

Comment:
We have no way of getting back to HOLD or knowing the previous state. 

I've discussed with Zac they can move it into ACTIVE before going to CORRECTION.